scripts/pipeline/orthology_benchmark.py

```diff
@@ -157,12 +160,49 @@ def get_core_names(species_names: List[str], host: str, port: int, user: str) ->
     return core_names
 
 
-def prep_input_for_orth_tools():
-    """Docstring"""
+def prep_input_for_orth_tools(source_dir: str, target_dir_ofinder: str) -> None:
```

Comment:
Can we just name all `target_dir_ofinder` `target_dir`? For a while, except that I realised it was consistent, my eyes kept telling me it was a typo and the intended was `target_dir_orthofinder`. However, since we know this script, aptly named `orthology_benchmark.py` is specific to orthofinder preparation, and for consistency with `source_dir` - `target_dir` makes good sense :+1:

Reply:
I'd be up for `target_dir_orthofinder` but I thought it might be too long. The script `orthology_benchmark.py` is not specific to OrthoFinder and as I said in the description, `prep_input_for_orth_tools()` and `run_orthology_tools()` will be extended to handle other tools. What I didn't say is that will happen already in Dec/Jan 😄

Reply:
This is really confusing, but I don't know the future! So the function will be used for other tools, but it has an orthofinder specific parameter passed in? Are you planning to then have a different parameter in the function for each of the other orthology-tools? And what of those that all take a simple `target_dir` of fasta files? To me it makes more sense to write this function more generically, and then it can be added to/altered to fit the next stages. It's nice to have a script that looks like it could be finished, without lots of silent placeholders no-one else knows about. If you were to win the lottery and decided to buy a tropical island without internet to live on for a year, someone else should be able to pick up the script, and add the functionality. What input do the other orthology-tools you have in mind take?

Reply:
I should've linked tickets for preparing input for other tools to the ticket ENSCOMPARASW-4952 so it'd make more sense for people not working on the project.

Yes, there will be input parameters for other tools. E.g. OMA will need `parameters.drw` file. I remember it requiring placing input fasta files in a particular location with respect to runnables and `parameters.drw`.

Reply:
For this, I would keep generic parameters shared by the tools, such as a `source_dir` (as you have) and a `target_dir` :point_left: because this is a common shared parameter that the majority of the tools, if not all of them, will use however you look at it, even if they require further processing in the prep stage. And for additional parameters that some tools do or do not have, perhaps you could incorporate those as and when needed in the future - although I suspect that might be more appropriate for the `run_orthology_tools`